spec/rails_helper.rb

```diff
@@ -1,9 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV["RAILS_ENV"] ||= 'test'
-require 'spec_helper'
+
```

Comment:
@matthewhuff89 @kaylorburdell A little wary of deleting requires from Rails files. I'd put it back for now.